### PR TITLE
Migrate to createPlugin api

### LINKED ISSUE #46: Migrate fusion-plugin-i18n to createPlugin

### PR DESCRIPTION
**Pull request by ganemone**
Fixes #46

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,18 +110,18 @@ import {FetchToken} from 'fusion-tokens';
...
-  : app.configure(FetchToken, fetch);
+  ? app.register(I18nLoaderToken, I18nLoader);
+  : app.register(FetchToken, fetch);
 ```
 
 - `I18n` - the core I18n library
 - `I18nLoader: (ctx: Context) => ({locale: string, translations: Object<string, string>})` - A function that provides translations
   - `ctx: {headers: {'accept-language': string}}` - A Koa context object
-- `fetch` - a [`fetch`](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API) implementation
+- `fetch` - a [`fetch`](https://developer.mozilla.org/en_US/docs/Web/API/Fetch_API) implementation
```
Comment (lhorie) [inline README.md]: broke the link

File: README.md [hunk 2]
```diff
@@ -72,50 +65,34 @@ import App from 'fusion-react';
...
 
   app.register(I18nToken, I18n);
   __NODE__
-    ? app.configure(I18nLoaderToken, I18nLoader);
-    : app.configure(FetchToken, fetch);
+    ? app.register(I18nLoaderToken, I18nLoader);
+    : app.register(FetchToken, fetch);
 
-  app.register(Hello);
+  // ....
```
Comment (lhorie) [inline README.md]: This change shouldn't remove the part of the example that shows how to actually use the service

Comment (ganemone) [reply README.md]: There are still examples in the readme

**Comment by ganemone** [comment]
!merge

**Pull request merged**